java/test/jmri/jmrix/cmri/serial/networkdriver/configurexml/ConnectionConfigXmlTest.java

```diff
@@ -46,7 +46,5 @@ public void tearDown() {
     @Override
     public void getInstanceTest() {
         super.getInstanceTest();
```

Comment:
Since you removed what made this test different than the overridden method, this method should be removed